games-strategy/wesnoth/wesnoth-1.18.2.recipe

```diff
@@ -0,0 +1,175 @@
+maps, and write your own scenarios or even full-blown campaigns. You can also \
+challenge up to eight friends—or strangers—and fight in epic multiplayer \
+fantasy battles."
+HOMEPAGE="http://wesnoth.org/"
+COPYRIGHT="2003-2024 The Battle for Wesnoth Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/wesnoth/files/wesnoth-1.18/wesnoth-$portVersion/wesnoth-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5f9f6d6105d48cf614eb06cf72273d62931dcb5d4982d75e851c3fa2fa70553a"
+PATCHES="wesnoth-$portVersion.patchset"
+ADDITIONAL_FILES="
+	wesnoth.rdef.in
+	wesnoth-postinstall.sh
+	"
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/wesnoth-postinstall.sh"
+
+ARCHITECTURES="?all !x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wesnoth$secondaryArchSuffix = $portVersion
+	cmd:wesnoth
+	cmd:wesnothd
+	cmd:wesnoth_campaignd
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_coroutine$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_graph$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_locale$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_regex$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libhistory$secondaryArchSuffix # Optional, for Lua console support
+	lib:libicudata$secondaryArchSuffix
+	lib:libicui18n$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_image_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	wesnoth_data == $portVersion
+	"
+
+SUMMARY_data="Battle for Wesnoth - Data files"
+ARCHITECTURES_data="any"
+
+PROVIDES_data="
+	wesnoth_data = $portVersion
+	"
+REQUIRES_data="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cairo1.18${secondaryArchSuffix}_devel # do not use xcairo
+	gcc_syslibs${secondaryArchSuffix}_devel
+	devel:libboost_coroutine$secondaryArchSuffix >= 1.83.0
+	devel:libboost_filesystem$secondaryArchSuffix >= 1.83.0
+	devel:libboost_graph$secondaryArchSuffix >= 1.83.0
+	devel:libboost_iostreams$secondaryArchSuffix >= 1.83.0
+	devel:libboost_locale$secondaryArchSuffix >= 1.83.0
+	devel:libboost_program_options$secondaryArchSuffix >= 1.83.0
+	devel:libboost_regex$secondaryArchSuffix >= 1.83.0
+	devel:libboost_system$secondaryArchSuffix >= 1.83.0
+	devel:libcurl$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libhistory$secondaryArchSuffix # Optional, for Lua console support
+	devel:libicuuc$secondaryArchSuffix >= 74
```

Comment:
Thanks for this, although it doesn't seem required when building, it forces the ICU version in REQUIRES. :)

Reply:
lol, should I remove it then?

Reply:
NO! :) just mentioning that it "forces" the correct ICU version, atleast in the end if need be we can force another version and see where we can disable the old(er) version when the time comes. :)

Reply:
Ah, ok, something to keep in mind for the future :)